src/crawlers/cheerio_crawler.js

```diff
@@ -365,7 +364,12 @@ class CheerioCrawler {
         this.supportedMimeTypes = new Set(DEFAULT_MIME_TYPES);
         if (additionalMimeTypes.length) this._extendSupportedMimeTypes(additionalMimeTypes);
 
-        this.requestOptions = requestOptions;
+
+        if (requestOptions) {
```

Comment:
This deprecation would deserve a mention in CHANGELOG

Reply:
BTW I didn't find any docs for transformRequestFunction

Reply:
Thanks for noticing. That's a misnaming. It's `prepareRequestFunction`